scripts/tools/linux-install-protoc

```diff
@@ -0,0 +1,35 @@
+#!/bin/bash
+
+if [[ "$(uname)" != "Linux" ]]; then
+  echo "Run on Linux only."
+  exit 1
+fi
+
+DIR="$(cd "$(dirname "${BASH_SOURCE[0]}")/.." && pwd)"
+. ${DIR}/all-utilities || { echo "Cannot load Bash utilities" ; exit 1 ; }
+
+PROTOC_VERSION="3.5.0"
+PROTOC_ARCHIVE="protoc-${PROTOC_VERSION}-linux-x86_64.zip"
+PROTOC_BASE_URL="https://github.com/google/protobuf/releases/download/v${PROTOC_VERSION}"
+PROTOC_URL="${PROTOC_BASE_URL}/${PROTOC_ARCHIVE}"
+PROTOC_BINARY='protoc'
+PROTOC_DIRECTORY="${TOOLS_DIR}/protoc"
+
+function update_protoc() {
+  local protoc_current="$(protoc --version | grep -oh '[0-9]*\.[0-9]*\.[0-9]*')" \
```

Comment:
The protoc_current doesn't seem to be used anywhere.  I'm guessing this was derived from the repo script, which doesn't have a version.  Other scripts would wrap much of the functionality that follows within a:

if [[ "${PROTOC_VERSION}" != "${protoc_current}" ]]; then

Reply:
even golang install script is the same

Reply:
I don't quite understand the comment.  The go install script has a:

if [[ "${GO_VERSION}" != "${go_current}" ]]; then

and if I run "grep _current *" in test-infra's scripts/tool/ then the only file that's currently not using the result of XXX_current is the one for repo.  I don't know how important this optimization is for the update function to no-op when the version is already the desired one, but in any case it seems like this protoc_current should either have something that reads/uses it or not exist at all.

Reply:
fixed